### ISSUE TITLE
Dev: Unittest failed on Windows OS cmd terminal but passed on VSCode IDE

### ISSUE THREAD

**Issue by mtreacy002** [anitab-org/bridge-in-tech-backend#133]
**Describe the bug**
Contributors using Windows OS cmd (and perhaps Linux OS) will see tests as failed when running `python -m unittest discover tests` from the CMD terminal. However the tests will pass on the VSCode IDE on Windows and Linux machines. The same tests will pass on both terminal and VSCodee IDE on a MacOS machines

**To Reproduce**
On Windows OS:
1. Go to VSCode Test explorer
2. Run all tests and see all tests passed
3. Now run 'python -m unittest discover tests' from CMD terminal
4. See all tests failed with error message `Can't load plugin: sqlalchemy.dialects:None

**Expected behavior**
Running unittest from the terminal should also pass so they are consistent with successful tests on VSCode IDE

**Screenshots**
On Windows OS:
<img width="958" alt="unittest errors" src="https://user-images.githubusercontent.com/29667122/92840236-34335200-f424-11ea-9e75-c22e3debfb13.png">

On Mac OS:
<img width="1680" alt="Screen Shot 2020-09-11 at 11 13 17 am" src="https://user-images.githubusercontent.com/29667122/92836552-e0bf0500-f41f-11ea-8d83-6bc244d8abd9.png">

**Desktop (please complete the following information):**
 - OS: Windows and Mac OS
 - Browser NA (VSCode  IDE)
 - Version ??
**Additional context**
Potential cause is the sqlalchemy.dialects.postgresql did not get picked up as plugin inside CMD terrminal on Windows. I have tried running `import sqlalchemy.dialects.postgresql` from python shell to check if the plugin is within the package. If the plugin is not included it should give error, but instead the result shows that plugin for postgresql dialect already installed, just didn't get picked up when running the unittest (see the orange box showing error message on Windows screenshot above).
* screenshot running import dialect passed without error
<img width="500" alt="postgre dialect" src="https://user-images.githubusercontent.com/29667122/92837787-57a8cd80-f421-11ea-8586-02ac0a87c62f.png">

**Comment by ashwinijha6** [comment]
@mtreacy002 is this issue available?

**Comment by mtreacy002** [comment]
Yes, @ashwinijha6  it's still available. Would you like to work on it?

**Comment by ashwinijha6** [comment]
@mtreacy002 is this just for windows OS or Can I check the bug for Linux OS also?

**Comment by mtreacy002** [comment]
@ashwinijha6 , I'm not sure to whether or not this also the case on Linux since I'm only testing this on Windows. What machine you're going to use to work on this issue?

**Comment by ashwinijha6** [comment]
> @ashwinijha6 , I'm not sure to whether or not this also the case on Linux since I'm only testing this on Windows. What machine you're going to use to work on this issue?

Hey @mtreacy002 I'll be using Linux OS (Ubuntu 16.04) for trying the issue. Could u assign it to me?

**Comment by mtreacy002** [comment]
Sure, @ashwinijha6 . will assign you to this now. 😉

**Comment by mtreacy002** [comment]
@ashwinijha6 , any update on this? Have you tried running on Linux? Did you face the same issue?

**Comment by ashwinijha6** [comment]
@mtreacy002 I need a little time on this as I'm facing an issue with pgadmin4.

**Comment by mtreacy002** [comment]
Sure, @ashwinijha6 . Let me know if you need help.

**Comment by mtreacy002** [comment]
@ashwinijha6 , can you please give update on this?

**Comment by ashwinijha6** [comment]
Yes @mtreacy002 the tests fail when performing them on the terminal
![bit_test](https://user-images.githubusercontent.com/45794450/110251152-34f90b00-7fa5-11eb-8b89-9ee738585e06.JPG)

**Comment by mtreacy002** [comment]
Ahhh, I see. @ashwinijha6, can you please add this point in the issue description (that same issue also happens on Linux OS and please indicate your terminal shell)? Are you able to  continue working on it for the fix?

**Comment by ashwinijha6** [comment]
> Ahhh, I see. @ashwinijha6, can you please add this point in the issue description (that same issue also happens on Linux OS and please indicate your terminal shell)? Are you able to continue working on it for the fix?

Ya sure @mtreacy002 I will add it in the description. Also I will continue working on the fix 👍

**Comment by mtreacy002** [comment]
@ashwinijha6 , can you please give us an update on this?

**Comment by ashwinijha6** [comment]
Yes @mtreacy002 I thought that it as an error due to sqlalchemy I tried installing by `pip install --upgrade sqlalchemy` (screenshot attached) but while running the tests I get the same error 
![sqlalchemy](https://user-images.githubusercontent.com/45794450/111037729-76793280-844b-11eb-9595-bb952320c014.JPG)
Could you help me with the python version that is being used for the project ? I'm using python3.7

**Comment by mtreacy002** [comment]
@ashwinijha6 , I'm also using Python 3.7 (3.7.9 to be exact). Btw, I noticed the dialect being imported is sqlite. It should be psycopg2.

**Comment by ashwinijha6** [comment]
> @ashwinijha6 , I'm also using Python 3.7 (3.7.9 to be exact). Btw, I noticed the dialect being imported is sqlite. It should be psycopg2.

ok yes I will check with that @mtreacy002

**Comment by Rahulm2310** [comment]
@ashwinijha6 any updates on this ?

**Comment by ashwinijha6** [comment]
> @ashwinijha6 any updates on this ?

I'm a bit preoccupied at moment @Rahulm2310, If you want you can assign it to someone else. Sorry for the delay

**Comment by Amulya-coder** [comment]
No issues @ashwinijha6 Thanks for giving the updates now it's open for OSH aspirants 😉.
